source/_includes/installation/container/cli.md

```diff
@@ -10,7 +10,7 @@
       --restart=unless-stopped \
       -e TZ=MY_TIME_ZONE \
       -v /PATH_TO_YOUR_CONFIG:/config \
-      --network=host \
+      -p 8123:8123 \
```

Comment:
Running Home Assistant is only support in host mode.

This change is thus not supported, and should therefore not be used or documented as such.

../Frenck